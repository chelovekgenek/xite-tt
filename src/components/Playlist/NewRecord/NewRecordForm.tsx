import React from "react"
import { Form } from "formik"

import { Input } from "components/commons/form"

export const NewRecordForm = () => (
  <Form>
    <Input label="Title" name="title" />
    <Input label="Artist" name="artist" />
    <Input label="Video URL" name="src" />
  </Form>
)
