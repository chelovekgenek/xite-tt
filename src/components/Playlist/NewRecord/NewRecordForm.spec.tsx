import React from "react"
import { shallow } from "enzyme"

import { NewRecordForm } from "./NewRecordForm"

describe("App", () => {
  it("renders correctly", () => {
    const comp = shallow(<NewRecordForm />)
    expect(comp).toMatchSnapshot()
  })
})
