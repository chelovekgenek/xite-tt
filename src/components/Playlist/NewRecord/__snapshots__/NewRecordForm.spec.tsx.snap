// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<Form>
  <Input
    label="Title"
    name="title"
  />
  <Input
    label="Artist"
    name="artist"
  />
  <Input
    label="Video URL"
    name="src"
  />
</Form>
`;
